app/frontend/src/features/connections/ConnectionsPage.tsx

```diff
@@ -0,0 +1,28 @@
+import { TabContext, TabPanel } from "@material-ui/lab";
+import React, { useState } from "react";
+import PageTitle from "../../components/PageTitle";
+import FriendsTab from "./friends/FriendsTab";
+import TabBar from "../../components/TabBar";
+
+const labels = {
+  TAB_ALL: "All",
+  TAB_FRIENDS: "Friends",
+};
+
+function ConnectionsPage() {
+  const [value, setValue] = useState<keyof typeof labels>("TAB_ALL");
+  return (
+    <>
+      <PageTitle>My Connections</PageTitle>
+      <TabContext value={value}>
+        <TabBar value={value} setValue={setValue} labels={labels} />
+        <TabPanel value="TAB_ALL">ALL</TabPanel>
+        <TabPanel value="TAB_FRIENDS">
+          <FriendsTab />
+        </TabPanel>
+      </TabContext>
```

Comment:
Is there an easy way to make tabs "routes"? Maybe a `<Switch>` instead of the tab panels? Would probably need a `useEffect` to set `value` too. This is also the case for the messages page, later.

Reply:
This was copied/adapted from the messages stuff which looks not too different from the example code in the Material UI page, so not sure without playing around with it a bit more tbh.

I did think that was annoying for messages though as currently there's no way to copy/paste a link that would get you straight to the group chat, but that might be for a separate PR/issue.

Reply:
Addressed this for the friend list tab by using URL params to manage state. The cool thing about that is that if the user visits a specific URL, it will jump straight to that tab rather than always defaulting to the first tab.